src/components/stepper/stepper.st.css

```diff
@@ -50,17 +52,3 @@
 .control:hover {
     color: value(PR3);
 }
-
-.up {
```

Comment:
You have removed .up and .down completely... this will remove them from the API (but we need them to access the buttons.

Please leave the classes and just have them empty if needed like this:
```
.up {
}

.down {
}
```

Reply:
agree